packages/web/src/modules/zoom/components/controls/host-control.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+
+import { ui } from '@application';
+import { Components } from '@types';
+
+import { ZoomContextType, useZoomContext } from '../../context';
+import { authorizeZoom } from '../../services';
+
+const { Button, Form } = ui;
+
+interface ZoomHostData {
+  meetingNumber: string;
+  password: string;
+}
+
+const ZoomHostControl: Components['ZoomHostControl'] = () => {
+  const zoomContext: ZoomContextType = useZoomContext();
+
+  const onSubmit = useCallback(
+    ({ meetingNumber, password }: ZoomHostData) => {
+      zoomContext.updateContext((prevState: ZoomContextType) => ({
+        ...prevState,
+        meetingNumber: meetingNumber.replaceAll(' ', ''),
+        password,
+      }));
+    },
+    [zoomContext],
+  );
+
+  return (
+    <>
+      {zoomContext.meetingNumber === '' &&
```

Comment:
I don't like empty string comparison. Create a local service `hasMeetingNumber` or something like that that will do the comparison and use it all around.